modules/discovery/info_disclosure/interesting_files.py

```diff
@@ -1,30 +1,44 @@
 from recon.core.module import BaseModule
-import warnings
+import csv
 import gzip
+import os
+import warnings
 from io import BytesIO
 
+
 class Module(BaseModule):
 
     meta = {
         'name': 'Interesting File Finder',
         'author': 'Tim Tomes (@lanmaster53), thrapt (thrapt@gmail.com), Jay Turla (@shipcod3), and Mark Jeffery',
-        'version': '1.1',
+        'version': '1.2',
         'description': 'Checks hosts for interesting files in predictable locations.',
         'comments': (
             'Files: robots.txt, sitemap.xml, sitemap.xml.gz, crossdomain.xml, phpinfo.php, test.php, elmah.axd, server-status, jmx-console/, admin-console/, web-console/',
+            f'CSV Default: {os.path.join(BaseModule.data_path, "interesting_files_verify.csv")}',
             'Google Dorks:',
             '\tinurl:robots.txt ext:txt',
             '\tinurl:elmah.axd ext:axd intitle:"Error log for"',
             '\tinurl:server-status "Apache Status"',
         ),
         'query': 'SELECT DISTINCT host FROM hosts WHERE host IS NOT NULL',
         'options': (
+            ('csv_file', os.path.join(BaseModule.data_path, 'interesting_files_verify.csv'),
+             True, 'custom filename map'),
             ('download', True, True, 'download discovered files'),
             ('protocol', 'http', True, 'request protocol'),
             ('port', 80, True, 'request port'),
         ),
+        'files': ['interesting_files_verify.csv'],
     }
 
+    def read_filenames_csv(self):
+        with open(os.path.expanduser(self.options['csv_file'])) as csvfile:
```

Comment:
You shouldn't need the `expanduser` call here because you took care of any pathing issues with `os.path.join(BaseModule.data_path, 'interesting_files_verify.csv')`. `with open(self.options['csv_file']) as csvfile:` should work fine. Please test.

Reply:
just confirmed that if I omit the expanduser I get the following:

```
[!] [Errno 2] No such file or directory: '~/foo.csv'.
```

Hit that before which is why I added that.

Reply:
obviously only comes into play when specifying the custom file... 
This is what my options look like in this test case:

```
Options:
  Name      Current Value  Required  Description
  --------  -------------  --------  -----------
  CSV_FILE  ~/foo.csv      yes       custom filename map
  DOWNLOAD  True           yes       download discovered files
  PORT      443            yes       request port
  PROTOCOL  https          yes       request protocol
  SOURCE    google.com     yes       source of input (see 'info' for details)
```

Certainly wanna support that shorthand if ppl are referencing their home dir.  The BaseModule.data_path only comes into play for the default

Reply:
Right... hm... now I want to do that for all of uses of data files. Mind creating an issue for that? I'll go ahead and approve this now. Nice work! Thank you for your contribution.

Reply:
For sure! Thanks so much for your prompt / thoughtful review.
Will open that issue now. Looking forward to diving into more of the code as I use the project more; really appreciate you writing it! love the interface and functionality; and the code is very approachable. Cheers

Reply:
Comments like this validate my time and effort. But also don't forget that the framework is useless without the modules that the community writes to leverage it. Thank you!